File: app/src/main/java/com/mgaetan89/showsrage/fragment/SettingsExperimentalFeaturesFragment.java
package com.mgaetan89.showsrage.fragment;

import com.mgaetan89.showsrage.R;

public class SettingsExperimentalFeaturesFragment extends SettingsFragment {
	@Override
	protected int getTitleResourceId() {
		return R.string.experimental_features;
	}

	@Override
	protected int getXmlResourceFile() {
		return R.xml.settings_experimental_features;
	}
}

File: app/src/main/kotlin/com/mgaetan89/showsrage/fragment/SettingsExperimentalFeaturesFragment.kt
package com.mgaetan89.showsrage.fragment

import com.mgaetan89.showsrage.R

class SettingsExperimentalFeaturesFragment : SettingsFragment() {
    override fun getTitleResourceId() = R.string.experimental_features

    override fun getXmlResourceFile() = R.xml.settings_experimental_features
}
